timemachine/cpp/src/nonbonded_all_pairs.cu

```diff
@@ -238,7 +171,7 @@ void NonbondedAllPairs<RealType>::execute_device(
 
     // (ytz) the nonbonded algorithm proceeds as follows:
 
-    // (done in constructor), construct a hilbert curve mapping each of the HILBERT_GRID_DIM x HILBERT_GRID_DIM x HILBERT_GRID_DIM cells into an index.
+    // (done in constructor), construct a hilbert curve sorting.
```

Comment:
I found the original comment useful. Is this description replicated somewhere else?

Reply:
Will add back in the comment in the hilbert constructor

Reply:
Added back in [3e0341c](https://github.com/proteneer/timemachine/pull/1113/commits/3e0341cba746d8cf46706d888db67e8c28823f5b)